libs/block-scripts/shared/generate-dist-block-metadata.js

```diff
@@ -27,6 +27,9 @@ export const generateDistBlockMetadata = async (extra) => {
     ? await fs.readJson(variantsJsonPath)
     : undefined;
 
+  const schema = blockprotocol.blockEntityType;
```

Comment:
Interesting, I'd actually thought about this and forgot to ask.

Do we maybe just want to stop spreading `...blockprotocol` and be explicit about what we do want to appear in `block-metadata.json`?

Unless we validate it doesn't have extra properties, users can put whatever they want in the object

Reply:
We could do, but they can put what they want in `block-metadata.json` after it's generated anyway. We only take the properties we want when populating the metadata that we save when a block is published.

Reply:
I guess it just feels a little strange to single out this specific property to clean-up. But I appreciate that's because it's *guaranteed* to be messy every time

Reply:
Yes – this is something we have introduced. I'm happy to update to remove the spread when we're next updating this but would rather not hold it up for that as we'd need to be careful we got it right